subt/rosbag2log.py

```diff
@@ -180,16 +180,21 @@ def extract_log(gen, out_name, append=False):
         with tarfile.open(args.filename, "r") as tar:
             for member in tar.getmembers():
                 if member.name.startswith('robot_data_0.bag'):
+                    print(member.name, "->", name)
                     f = tar.extractfile(member)
-                    extract_log(read_rosbag_fd_gen(f), out_name)
+                    extract_log(read_rosbag_fd_gen(f, verbose=args.verbose), out_name, verbose=args.verbose)
                     robot_data_0_processed = True
-                elif member.name.startswith('robot_data_1.bag') and args.all:
+                elif member.name.startswith('robot_data_1.bag'):
```

Comment:
this could be actually "a bug" from old times ... i.e. there is a chance that also `robot_data_2.bag` exists (the splitting is somehow strange and we may push more than 2GB data
... probably ignore this note, I was just wondering why we do care about `_1.bag` ;) ...

Reply:
I am actually testing if the log cannot be 3GB which would give us `robot_data_0.bag`, `robot_data_1.bag` and `robot_data_2.bag.active`. I think it just might work. We will see.